### DISCUSSION TITLE
Consider useSwipeable hook

### DISCUSSION THREAD

**Discussion by steveoh** [adobe/react-spectrum#6798] [Ideas]
I see there is a [swipeable](https://react-spectrum.adobe.com/react-aria/examples/swipeable-tabs.html) tab example. Could this behavior be extracted as a hook for react aria? It has utility for popovers and other components. 

Do gestures fit into the scope of these packages or is the general suggestion to use existing work like react-swipeable?

**Comment by snowystinger** [comment]
That's not in our scope currently, but I can't say that it's because we're purposefully staying away from it. I think it's more that we haven't needed it ourselves. Generally we create our own hooks for aria patterns or interactions which need cross browser/device normalization. So if there is a compelling case for this, it could be considered. I'd start by following https://react-spectrum.adobe.com/contribute.html#feature-requests
